### ISSUE TITLE
using any() on a logical column in tbl_df fails

### ISSUE THREAD

**Issue by yannabraham** [yannabraham/Radviz#4]
quick reproducible example:

```
x <- tbl_df(data.frame(rvalid=sample(c(TRUE,FALSE),100,replace=TRUE)))
any(x[,"rvalid"]) # will throw an error
any(data.frame(x)[,"rvalid"]) # will work
any(x[,"rvalid"]==TRUE) # will work too
```

Issue has been filled with the `tibble` team

use any(x[,"rvalid"]==TRUE) in do.radviz for robustness

**Comment by yannabraham** [comment]
suggestion from tibble team is to use `x[['rvalid']]`

**Comment by yannabraham** [comment]
bug fixed with c964affa58f1de56376977bc33e0cf69aa00228d
